lesson08/assignment/video_cli/l8/src/Home.vue

```diff
@@ -1,34 +1,12 @@
 <template>
     <div>
-      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-        <a class="navbar-brand" href="#">My Website</a>
-        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
-        <div class="collapse navbar-collapse" id="navbarNav">
-          <ul class="navbar-nav">
-            <li class="nav-item">
-              <a class="nav-link" href="index.html">Home Page</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="image-pg.html">Image Page</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="details.html">Top 3 Page</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" href="form.html">Contact Us</a>
-            </li>
-          </ul>
-        </div>
-      </nav>
   
       <div class="container">
         <h1>Welcome to my movie website! Search for a film here.</h1>
         <form id="movieForm" autocomplete="off">
           <div class="form-group">
             <label for="movie">Movie Name</label>
-            <input class="form-control" type="text" id="movie" placeholder="Movie" v-model="searchTerm">
+            <!-- <input class="form-control" type="text" id="movie" placeholder="Movie" v-model="searchTerm"> -->
```

Comment:
Just removed this as I was trying to get the pages to render.  You can uncomment this out and work on the API functionality.